gradle.properties

```diff
@@ -14,6 +14,14 @@ ROPSTEN_NETWORK_BACKEND_BASE_HOST=https://apichain-dev.blockchainds.com
 MAIN_NETWORK_BACKEND_BASE_HOST_WS=https://api.blockchainds.com
 ROPSTEN_NETWORK_BACKEND_BASE_HOST_WS=https://api-dev.blockchainds.com
 
+APPCOINS_ADVERTISEMENT_DEV_BIND_ACTION=com.appcoins.wallet.advertising.action.BIND
```

Comment:
Shouldn't this have .dev. as well? I recall that we've talked about this but can't remember the conclusion